contracts/libraries/PostLib.sol

```diff
@@ -389,8 +389,12 @@ library PostLib  {
             UserLib.Permission.NONE,
             false
         );
-
-        if (postContainer.info.rating > 0) {
+    
+        // DownvoteExpertPost = -1;
+        // UpvotedExpertPost = 5;
+        // + 1 upvote and 1 down vote -> postRating = 0 but userRating +4 (need fix?)
```

Comment:
Is rating for upvote the same as rating for downvote? I think nothing is needed in that case.